apps/parse/lib/parse/calendar_attributes.ex

```diff
@@ -0,0 +1,25 @@
+defmodule Parse.CalendarAttributes do
+  @moduledoc """
+  Parser for GTFS calendar_attributes.txt
+  """
+
+  use Parse.Simple
+
+  defstruct [
+    :service_id,
+    :description,
+    :schedule_name,
+    :schedule_type,
+    :schedule_typicality
+  ]
+
+  def parse_row(row) do
+    %__MODULE__{
+      service_id: :binary.copy(row["service_id"]),
+      description: :binary.copy(row["service_description"]),
+      schedule_name: :binary.copy(row["service_schedule_name"]),
+      schedule_type: :binary.copy(row["service_schedule_type"]),
+      schedule_typicality: row["service_schedule_typicality"]
+    }
```

Comment:
These will always end up as binaries: you probably need to do some conversions (empty binary -> nil, binary -> integer for typicality).

Reply:
Probably a test of `parse_row/1` would catch this.